node/node.go

```diff
@@ -159,11 +158,7 @@ func NewNode(ctx context.Context, conf config.NodeConfig, p2pKey crypto.PrivKey,
 		return nil, fmt.Errorf("couldn't get settlement client named '%s'", conf.SettlementLayer)
 	}
 	if conf.SettlementLayer == "mock" {
-		pubKeybytes, err := signingKey.GetPublic().Raw()
-		if err != nil {
-			return nil, err
-		}
-		conf.SettlementConfig.ProposerPubKey = hex.EncodeToString(pubKeybytes)
+		conf.SettlementConfig.KeyringHomeDir = conf.RootDir
```

Comment:
why do we need to initialize the `KeyringHomeDir` here  vs just taking it from the dymint.toml? we have `keyring_home_dir` in the dymint.toml.